src/main/kotlin/no/nav/syfo/kafka/recordprocessors/SpleisRecordProcessor.kt

```diff
@@ -32,9 +35,18 @@ class SpleisRecordProcessor(
             log.error("Exception in [$topicUtbetaling]-processor: $e", e)
         }
     }
-
     private fun processUtbetalingSpleisEvent(utbetaling: UtbetalingSpleis) {
-        utbetalingSpleisDAO.storeSpleisUtbetaling(utbetaling)
+        val utbetTom = calculateUtbetTom(utbetaling)
+        utbetalingSpleisDAO.storeSpleisUtbetaling(utbetaling, utbetTom)
+        if (utbetTom != LocalDate.parse(utbetaling.tom)) {
+            log.info(
```

Comment:
Logger litt info her for å høste erfaringer om hvordan det fungerer - kan fjernes etter hvert,